invites_test.go

```diff
@@ -0,0 +1,37 @@
+package rollbar
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestListInvites(t *testing.T) {
+	teardown := setup()
+	defer teardown()
+	handUrl := fmt.Sprintf("/team/%s/invites/", teamID)
+
+	mux.HandleFunc(handUrl, func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(http.StatusOK)
+
+		// Check if we've hit the second page and return an empty paginated
+		// response thus simulating the apis behaviour.
+		splStr := strings.Split(r.URL.RawQuery, "&")
+		if splStr[1] == "page=2" {
+			fmt.Fprint(w, fixture("teams/empty_invites.json"))
+		} else {
+			fmt.Fprint(w, fixture("teams/invites.json"))
+		}
+	})
+
+	// team_id should e int for this particular func
+	teamTOI, _ := strconv.Atoi(teamID)
+	_, err := client.ListInvites(teamTOI)
+
+	if err != nil {
+		t.Fatal(err)
+	}
+}
```

Comment:
Actually, we could even check the value of teamID and compare it to fixture's value.

Reply:
Well, this is listing all the invites. That means that there might be multiple team ids returned and many users which are attached to those teams. I don't think it makes sense to test that.